zh-CN/mapping_dev.zh_CN

```diff
@@ -250,33 +250,33 @@ dlgScMasterPasswordCreate.edt"Retype Again"=重新输入
 dlgScMasterPasswordCreate.hdl=配置加密密码存储的主密码
 dlgScMasterPasswordCreate.inf=主密码用于保护用于向服务器进行身份验证的密码和口令。
 dlgScMasterPasswordCreate.lbl"Not using a master password makes your passwords and passphrases readable for everyone who has access to the password file located at $1. Use this option only if you are sure that this file is safe."=\
- Not using a master password makes your passwords and passphrases readable for everyone who has access to the password file located at $1. Use this option only if you are sure that this file is safe.
+ 不使用主密码使您的密码和密码短语对访问位于 $1 的密码文件的每个人都是可读的。仅当您确定此文件安全时才使用此选项。
 dlgScMasterPasswordCreate.lbl"This master password is case-sensitive and should contain lowercase and uppercase characters, digits and other characters. Longer passwords are in general more secure than shorter ones."=\
- 此主密码区分大小写，应包含小写和大写字符，数字和其他字符。较长的密码通常比较短的密码更安全。
+ 此主密码区分大小写，应包含小写和大写字符、数字和其他字符。长密码通常比短密码更安全。
 dlgScMasterPasswordCreate.rbt"Don't use a master password"=不要使用主密码
 dlgScMasterPasswordCreate.rbt"Use the following master password"=使用以下主密码
 dlgScMasterPasswordCreate.tle=主密码
-dlgScPropertyEdit.edt"Value"=Value
-dlgScPropertyEdit.hdl=Edit low-level property value
-dlgScPropertyEdit.inf%1=Set the value for property '$1'
-dlgScPropertyEdit.rbt"false"=false
-dlgScPropertyEdit.rbt"true"=true
-dlgScPropertyEdit.tle=Edit Property
-dlgScRegisterFreeUpdatesExpiredLicense.btn"Purchase Update"=Purchase Update
-dlgScRegisterFreeUpdatesExpiredLicense.fur=You may use an older SmartGit version or purchase an update license.
-dlgScRegisterFreeUpdatesExpiredLicense.hdl=The free update period for this license does not cover this version.
-dlgScRegisterFreeUpdatesExpiredLicense.tle=SmartGit License
+dlgScPropertyEdit.edt"Value"=值
+dlgScPropertyEdit.hdl=编辑低级属性值
+dlgScPropertyEdit.inf%1=设置属性 “$1” 的值
+dlgScPropertyEdit.rbt"false"=错误
+dlgScPropertyEdit.rbt"true"=正确
+dlgScPropertyEdit.tle=编辑属性
+dlgScRegisterFreeUpdatesExpiredLicense.btn"Purchase Update"=购买更新
+dlgScRegisterFreeUpdatesExpiredLicense.fur=您可以使用旧的智能工具版本或购买更新许可证。
```

Comment:
"SmartGit" is product name, it should be keeped.

Reply:
Thanks! I've fixed it.